db/migrate/20150112152944_create_scaptimony_arf_report_raws.rb

```diff
@@ -1,7 +1,7 @@
 class CreateScaptimonyArfReportRaws < ActiveRecord::Migration
   def change
     create_table :scaptimony_arf_report_raws, :id => false do |t|
-      t.references :arf_report, :index => true, :null => false
+      t.references :arf_report, :null => false
```

Comment:
we create that index on line 8, no need for this (throws an exception because index already created)